### ISSUE TITLE
convert_to_svg should report any errors to the user

### ISSUE THREAD

**Issue by eulertour** [ManimCommunity/manim#621]
A lot of system-dependent things can go wrong while attempting to perform this conversion, but by suppressing the error we're making it harder for the user to debug it. Any error messages generated should be reported to the user in the following exception.

https://github.com/ManimCommunity/manim/blob/a1180be72d29adcf7d6a02c044e933acbf794d68/manim/utils/tex_file_writing.py#L196-L207

**Comment by kilacoda** [comment]
So we just have to remove lines 204-205?

**Comment by leotrs** [comment]
No. Instead of using `os.system` or `subprocess.run`, I think we want to use `subprocess.check_call`. Or is there a better alternative?

Note I tried this in #546 and it's a bit more complicated than it looked at first :sweat_smile:

**Comment by PhilippImhof** [comment]
This reminds me of https://github.com/ManimCommunity/manim/pull/561
